clients/fides-js/src/lib/consent-types.ts

```diff
@@ -85,6 +86,7 @@ export type PrivacyExperience = {
   tcf_systems?: Array<TCFVendorRecord>;
   tcf_features?: Array<TCFFeatureRecord>;
   tcf_special_features?: Array<TCFFeatureRecord>;
+  gvl?: GVLJson;
```

Comment:
When is gvl undefined? When tcf mode is disabled?

Reply:
yes, that's right!